python/lsst/pipe/tasks/makeCoaddTempExp.py

```diff
@@ -164,8 +183,22 @@ def createTempExp(self, calexpRefList, skyInfo, visitId=0):
                 calExpRef = calExpRef.butlerSubset.butler.dataRef("calexp", dataId=calExpRef.dataId,
                                                                   tract=skyInfo.tractInfo.getId())
                 calExp = self.getCalExp(calExpRef, bgSubtracted=self.config.bgSubtracted)
-                exposure = self.warpAndPsfMatch.run(calExp, modelPsf=modelPsf, wcs=skyInfo.wcs,
-                                                    maxBBox=skyInfo.bbox).exposure
+                covName = 'covar'
+                for key, val in calExpRef.dataId.items():
+                    covName += '_%s_%s'%(key, val)
```

Comment:
I don't actually see this being used anywhere.  Did you intend to write the covariance out with the butler, and then decide not to?  I do think that'd be worth supporting, as it's a lot easier to explore the results after the processing is complete, especially if the processing is at all slow.  And I think you'll need it anyway when you move on to coadding the covariances.